src/main/java/com/github/wovnio/wovnjava/WovnServletFilter.java

```diff
@@ -1,33 +1,112 @@
 package com.github.wovnio.wovnjava;
 
 import java.io.IOException;
+import java.io.PrintWriter;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
 
 import javax.servlet.Filter;
 import javax.servlet.FilterChain;
 import javax.servlet.FilterConfig;
 import javax.servlet.ServletException;
 import javax.servlet.ServletRequest;
 import javax.servlet.ServletResponse;
+import javax.servlet.ServletOutputStream;
 import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
 
 import org.apache.commons.logging.Log;
 import org.apache.commons.logging.LogFactory;
 
 public class WovnServletFilter implements Filter {
-    protected Interceptor interceptor;
+    //protected Interceptor interceptor;
+    private Settings settings;
 
-    public static final String VERSION = "0.2.1";
+    public static final String VERSION = Settings.VERSION;  // for backword compatibility
```

Comment:
this is not used

Reply:
But the class and field is public.
So user program can access it.